wiki/Beatmap_Editor/Song_Setup/en.md

```diff
@@ -1,145 +1,151 @@
-
--   **Artist:** **The band, singer, or group that performed or created the song.** osu! will pull the artist from the mp3 file, but please double check it to make sure it is spelled and punctuated correctly. **Anime/Game titles or companies do not belong here (they belong to Tags).** You can usually find the composer for anime and game songs on Wikipedia or other reliable sources such as [AnimeNewsNetwork](https://www.animenewsnetwork.com/) or [MyAnimeList](https://myanimelist.net/). Foreign names should be presented in the same name order as used on Wikipedia (generally, Given Name followed by Family Name, the western order, for consistency).
-
-<!-- -->
-
--   **Romanised Artist:** **The romanised (English) name of the artist of the song.** It should be presented in the same name order as used on Wikipedia (generally, Given Name followed by Family Name, the western order, for consistency). This field will become active if you use Unicode characters in the *Artist* field.
-
-<!-- -->
+# Song Setup
 
--   **Title:** **The name of the song.** It will be pulled automatically from the mp3 file, but double check for spelling or capitalization errors, especially if you get the mp3 online.
+![Where the Song Setup is located in the Beatmap Editor][SS_nav]
 
-<!-- -->
+Song Setup is an important screen in [beatmapping][Beatmapping]. It contains metadata and general beatmap configuration settings for the beatmap.
 
--   **Romanised Title:** **The romanised (English) name of the name of the song.** Again, this field will become active if you use Unicode characters in the *Title* field.
+## General
 
-<!-- -->
-
--   **Beatmap Creator:** **That's you.** osu! should handle this automatically, so you shouldn't edit this. If you are including a [guest difficulty](/wiki/Glossary) made by someone else, their name belongs in the Difficulty field below, not here.
-
-<!-- -->
-
--   **Difficulty:** **The name of the difficulty of map.** The dropdown contains the preset names: Easy, Normal, Hard, Insane. You can select one of the default names, or create your own name. While it's good to be creative, try to make it very clear which one is harder than others as ambiguous difficulty names can annoy players. This would also be the field where you indicate a guest mapper, if this is their difficulty (e.g. "Larto's Hard").
+### Song and Map Meta Data
 
-<!-- -->
+![An example of complete data (General tab)][SS_gen]
 
--   **Source:** **Where this song originate from.** This field should generally only be included for Anime or Video Game songs, and occasionally for Novelty (Movies, TV, or Internet) songs. If the song is from an anime or video game, or is famous for its use as a TV or movie theme, the title of that source goes here. Rock and Pop songs should generally leave the field blank. **This is not to be used for things like album titles.**
+[Guide on finding Metadata][forum_metadata].
```

Comment:
Move this up and change it to `_See also: [\[Guide\] Metadata](https://osu.ppy.sh/community/forums/topics/249288)_.`
.. I was going to port this guide like 2-3 months ago, but stuff happened with Guides so 🤷‍♂️